recipes/hazelcast-cpp-client/all/conanfile.py

```diff
@@ -44,7 +43,7 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("boost/1.79.0", transitive_headers=True, transitive_libs=True)
+        self.requires("boost/1.81.0", transitive_headers=True, transitive_libs=True)
```

Comment:
```suggestion
        self.requires("boost/1.83.0", transitive_headers=True, transitive_libs=True)
```

Reply:
It used to fail with boost 1.83.0 https://c3i.jfrog.io/c3i/misc/logs/pr/21754/3-linux-gcc/hazelcast-cpp-client/5.2.0//5d9d2780a74d4113b0223cf73d9be450956502e3-test.txt
But test program simplification may work around this error.